packages/valory/skills/market_manager_abci/behaviours.py

```diff
@@ -222,7 +222,7 @@ def get_active_bets(self) -> Generator[None, None, Dict[str, Dict[str, int]]]:
             self.synchronized_data.safe_contract_address.lower()
         )
         if user_positions is None:
-            return {}
+            return []
```

Comment:
Does not match the specified return type:

https://github.com/valory-xyz/trader/blob/f100392287ed1d58f9529aa59ffea23a260b313c/packages/valory/skills/market_manager_abci/behaviours.py#L213